src/libraries/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs

```diff
@@ -407,6 +407,12 @@ public static IEnumerable<object[]> Match_Basic_TestData()
             yield return new object[] { @".*\dfoo", "1fooThis1foo should 1foo match", RegexOptions.None, 4, 9, true, "This1foo" };
             yield return new object[] { @".*\dfoo", "This shouldn't match 1foo", RegexOptions.None, 0, 20, false, "" };
 
+            // Turkish case sensitivity
+            foreach (RegexOptions options in new[] { RegexOptions.None, RegexOptions.Compiled })
+            {
+                yield return new object[] { @"[\u0120-\u0130]", "\u0130", options, 0, 1, true, "\u0130" };
+            }
```

Comment:
Running each case with every available engine is handled elsewhere.
```suggestion
            yield return new object[] { @"[\u0120-\u0130]", "\u0130", RegexOptions.None, 0, 1, true, "\u0130" };
```